src/org/labkey/test/tests/ListArchiveExportTest.java

```diff
@@ -0,0 +1,124 @@
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.Locators;
+import org.labkey.test.categories.Daily;
+import org.labkey.test.categories.Hosting;
+import org.labkey.test.components.list.ManageListsGrid;
+import org.labkey.test.util.ApiPermissionsHelper;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.ListHelper;
+import org.labkey.test.util.PermissionsHelper;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+@Category({Daily.class, Hosting.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 2)
+public class ListArchiveExportTest extends BaseWebDriverTest
+{
+    private final static String LIST_SUBFOLDER = "List subfolder";
+    private final static String LIST_FOLDER_A = "LIST_FOLDER_A";
+    private final static String LIST_A = "List Export A";
+    private final static String LIST_B = "List Export B";
+    private final static String _listUser = "listuser@listarchiveexport.test";
+
+    @Override
+    protected @Nullable String getProjectName()
+    {
+        return getClass().getSimpleName() + " Project";
+    }
+
+    @Override
+    public List<String> getAssociatedModules()
+    {
+        return Arrays.asList("list");
+    }
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        ListArchiveExportTest initTest = (ListArchiveExportTest) getCurrentTest();
+        initTest.doSetUp();
+    }
+
+    private void doSetUp()
+    {
+        _containerHelper.createProject(getProjectName(), null);
+        createListWithData(LIST_B, Map.of("Shape", "Triangle", "Count", "4"));
+
+        _containerHelper.createSubfolder(getProjectName(), LIST_SUBFOLDER);
+        createListWithData(LIST_A, Map.of("Shape", "Square", "Count", "7"));
+
+        _containerHelper.createProject(LIST_FOLDER_A);
+        createListWithData(LIST_A, Map.of("Shape", "Circle", "Count", "10"));
+
+        _userHelper.createUser(_listUser);
+        ApiPermissionsHelper _permissionHelper = new ApiPermissionsHelper(LIST_FOLDER_A);
+        _permissionHelper.addMemberToRole(_listUser, "Folder Administrator", PermissionsHelper.MemberType.user);
+
+        _permissionHelper = new ApiPermissionsHelper(getProjectName());
+        _permissionHelper.addMemberToRole(_listUser, "Reader", PermissionsHelper.MemberType.user);
+    }
+
+    private void createListWithData(String name, Map rowData)
+    {
+        log("Creating test list in " + getCurrentContainerPath());
+        _listHelper.createList(getCurrentContainerPath(), name, ListHelper.ListColumnType.AutoInteger, "RowId",
+                new ListHelper.ListColumn("Shape", "Shape", ListHelper.ListColumnType.String),
+                new ListHelper.ListColumn("Count", "Count", ListHelper.ListColumnType.Integer));
+        _listHelper.insertNewRow(rowData);
+    }
```

Comment:
Use API helpers to avoid a bunch of unnecessary UI interaction.
```suggestion
    private void createListWithData(String name, Map rowData)
    {
        var connection = createDefaultConnection();
        var listDef = new IntListDefinition(name, "RowId").setFields(List.of(
                new FieldDefinition("Shape", FieldDefinition.ColumnType.String),
                new FieldDefinition("Count", FieldDefinition.ColumnType.Integer));
        var dataGenerator = listDef.create(connection, getCurrentContainerPath());
        dataGenerator.insertRows(connection, List.of(rowData));
    }
```
Also, `ListHelper.ListColumn` is deprecated and should never be used.